test/unittest/tests/wrappers/connection_wrapper.cpp

```diff
@@ -16,7 +16,18 @@
 ConnectionWrapper::ConnectionWrapper(const std::string &db_home)
     : _conn_impl(nullptr), _conn(nullptr), _db_home(db_home)
 {
-    utils::throwIfNonZero(mkdir(_db_home.c_str(), 0700));
+    struct stat sb;
+    /* Check if the DB Home exists and is a directory, without this the mkdir can fail on some
+     * platforms (win). */
```

Comment:
One last nitpick sorry, multiline comments are meant to have the start/end delimiters on separate lines, i.e.
```
/*
 * Check if the DB...
 */
```